components/gitpod-db/src/typeorm/entity/db-workspace.ts

```diff
@@ -31,6 +31,13 @@ export class DBWorkspace implements Workspace {
     @Index("ind_creationTime")
     creationTime: string;
 
+    @Column({
+        ...TypeORM.UUID_COLUMN_TYPE,
+        transformer: Transformer.MAP_NULL_TO_UNDEFINED,
+    })
+    @Index()
+    organizationId: string;
```

Comment:
Given the transformer maps NULL to Undefined, should the field be marked as Optional

```suggestion
    organizationId?: string;
```

Reply:
good catch! thank you